### ISSUE TITLE
Correct indentation of multiline ternary expressions in tagged template literals

### ISSUE THREAD

**Issue by cjhowe7** [eslint/eslint#9387]
**Tell us about your environment**
I am running eslint through Visual Studio code.

* **ESLint Version:** 4.8.0
* **Node Version:** 8.6.0
* **npm Version:** 5.3.0

**What parser (default, Babel-ESLint, etc.) are you using?**
default

**Please show your full configuration:**

<details>
<summary>Configuration</summary>

```js

  "eslintConfig": {
    "extends": "standard",
    "plugins": [
      "mocha"
    ],
    "env": {
      "mocha": true,
      "browser": true,
      "node": true
    },
    "parserOptions": {
      "ecmaVersion": 8
    }
  }

```

</details>

**What did you do? Please include the actual source code causing the issue.**

```js
  const renderProfile = () => {
    if (state.profile) {
      return html`
        <div>
          <p>
            <label>Name</label>
            <span>${state.profile.name}</span>
          </p>
          <p>
            <label>Email</label>
            <span>${state.profile.email}</span>
          </p>
          <p>
            <label>Major</label>
            ${state.profile.editing
    ? html`<input placeholder="${state.profile.major ||
                  'Undecided'}" type="text" />`
    : html`<span>${state.profile.major || 'Undecided'}</span>`}
          </p>
          ${state.profile.editing
    ? html`<button onclick=${() =>
      emit('submit-profile')}>Submit</button>`
    : html`<button onclick=${() => emit('edit-profile')}>Edit</button>`}
        </div>
      `
    } else {
      return html`
        <span class="${loading}">Loading...</span>
      `
    }
  }
```

**What did you expect to happen?**
The then and else clauses of the ternary expressions embedded inside the tagged template literals above should be indented past the conditional in the ternary expression when using `eslint --fix`. The then and else clauses should be indented two spaces past the conditional clause, which is the case when writing ternary expressions outside of tagged template literals.

**What actually happened? Please include the actual, raw output from ESLint.**
As can be seen above, the indentation is put at 4 spaces for the then and else clauses of the ternary expression, and an error is given if trying to indent these.

**Comment by cjhowe7** [comment]
Actually, this basically happens with any multiline expression embedded in a tagged template literal. I think this should be treated differently than a normal template literal.

**Comment by not-an-aardvark** [comment]
Thanks for the report. This is working as intended -- see https://github.com/eslint/eslint/issues/8880 and https://github.com/eslint/eslint/issues/8982.

**Comment by not-an-aardvark** [comment]
Closing because this is working as intended.
